src/cradle/rpclib/server/main.cpp

```diff
@@ -147,7 +147,7 @@ run_server(cli_options const& options)
 
     register_and_initialize_all_domains();
 
-    rpc::server srv(options.port);
+    rpc::server srv("127.0.0.1", options.port);
```

Comment:
Well I suppose we should first ask if it should be hard-coded like this. Setting the listening address to `127.0.0.1` means that the server will only listen for connections from within the local computer. This is necessary for normal Windows usage because otherwise Windows issues a warning about the application's network usage. I'm inclined to think the server should always work like this, at least for now. Any objections?

Reply:
No objections.